.github/workflows/build.yaml

```
@@ -9,14 +9,14 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        ghc: ['8.4.4', '8.6.5', '8.8.3', '8.10.2', '9.0.1', '9.2.1', '9.4.2', '9.6.3', '9.8.1']
+        ghc: ['8.4.4', '8.6.5', '8.8.4', '8.10.7', '9.0.1', '9.2.8', '9.4.8', '9.6.6', '9.8.2', '9.10.1']
```

Semi-related:
I haven't been keeping up with the ecosystem, but I think it might be fine to drop everything before the 9.x series.

```suggestion
        ghc: ['9.0.1', '9.2.8', '9.4.8', '9.6.6', '9.8.2', '9.10.1']
```

9.0.1 was released in Feb 2021, so 3 years doesn't feel like a bad cutoff for an ecosystem that moves pretty fast.

But I would appreciate thoughts from someone more involved.
What are your thoughts, @ysangkok?

---

I think it's nice to keep support if it doesn't cost anything and CI can still test it.

---

Fair. I'm mostly thinking of [hypothetical future] where supporting a really old version requires `#ifdef` and friends. Although since it's not a problem right now, that's moot for now.